cluster-autoscaler/cloudprovider/azure/azure_container_service_pool.go

```diff
@@ -96,7 +104,7 @@ func (agentPool *ContainerServiceAgentPool) GetPool(agentProfiles *[]containerse
 		for _, value := range *agentProfiles {
```

Comment:
Do we still need the check: "agentPool.serviceType == vmTypeACS" since we are already in the ACS path here ?

Reply:
hmm, yep, this is not required here

Reply:
Fixed now